### DISCUSSION TITLE
Management console throws JavaScript errors

### DISCUSSION THREAD

**Discussion by jenstroeger** [docker-library/rabbitmq#754] [General]
Using a pretty standard command line
```
docker run --hostname pooh-rabbit --name pooh-rabbit -p 5672:5672 -p 15672:15672 -e RABBITMQ_DEFAULT_USER=guest -e RABBITMQ_DEFAULT_PASS=guest rabbitmq:3-management
```
some parts of the management console show the following issues, e.g. `http://localhost:15672/#/exchanges` 

![Screenshot 2024-12-20 at 14 49 19](https://github.com/user-attachments/assets/4a9191b8-6815-4c10-be19-ee5657a2f54e)

or `http://localhost:15672/#/queues`

![Screenshot 2024-12-20 at 14 46 46](https://github.com/user-attachments/assets/8066dd5b-7e4d-4f05-b037-6f9b617f3ec6)

**Comment by michaelklishin** [comment]
@jenstroeger this question has been asked thousands of times and has nothing to do with the image, so I'm not sure why you have brought it up here and not in RabbitMQ's Discussions. 

It is [covered in the docs](https://www.rabbitmq.com/docs/upgrade#rabbitmq-plugins-compatibility). Clear your browser's cache.

In `4.0.x` — the only version [covered by community support](https://www.rabbitmq.com/release-information) — there is at least a partial solution https://github.com/rabbitmq/rabbitmq-server/pull/12779.
